src/RepairManager/actions/cordon_action.py

```diff
@@ -0,0 +1,22 @@
+import os, sys
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I don't think `sys.path.append` is need if the files to import are in the same directory